code/modules/mob/living/simple_animal/vore/carp.dm

```diff
@@ -19,6 +19,9 @@
 	pixel_x = -16
 	pixel_y = -16
 
+/mob/living/simple_animal/hostile/carp/large/huge
+	icon = 'icons/mob/vore64x64.dmi'
```

Comment:
I kind of don't really understand what this fixes. See line 4.

Reply:
large/huge has icon override in the polaris version of the large/huge mob.

Reply:
oh, rip